project/src/main/puzzle/piece/frame-input.gd

```diff
@@ -22,6 +22,8 @@ var _buffer: bool
 # If 'true' the inputs will be printed to the console. These printed inputs can be converted into an input replay.
 var _print_inputs := false
 
+onready var _buffer_timer: Timer = $BufferTimer
```

Comment:
Document this field

Reply:
Added comment, `# If this timer is active, the input was recently pressed and can be popped with pop_buffered_input`